src/leap/mail/imap/service/imap.py

```diff
@@ -146,16 +208,8 @@ def run_service(store, **kwargs):
     :returns: the port as returned by the reactor when starts listening, and
```

Comment:
docstring mentions store, maybe it's good to document `soledad_sessions` hear (and maybe is not needed to do it in `LeapIMAPFactory`).